Source/WebKit/Shared/cairo/ShareableBitmapCairo.cpp

```diff
@@ -43,7 +43,7 @@ void ShareableBitmap::validateConfiguration(Configuration&)
 {
 }
 
-CheckedUint32 ShareableBitmap::calculateBytesPerRow(WebCore::IntSize size, const Configuration&)
+CheckedUint32 ShareableBitmap::calculateBytesPerRow(WebCore::IntSize size, const ShareableBitmapConfiguration&)
```

Comment:
Same change is required on the `validateConfiguration()` and `calculateBytesPerPixel()` methods.